### DISCUSSION TITLE
Can I mock external content in an iframe?

### DISCUSSION THREAD

**Discussion by swfz** [mswjs/msw#952] [Q&A]
Is it possible to mock external content in an iframe or object tag in msw?

I was able to mock the image specified in src of img tag, so I thought I could do the same, but it didn't work.

Here is an example of what I want to do

````
rest.get("https://example.com/hoge.html", async(req, res, ctx) => {
    return res(ctx.status(200), ctx.xml('<html><div>hoge</div></html>'))
})
````

```
 <iframe src="https://example.com/hoge.html"></iframe>
```

**Comment by kettanaito** [comment]
Hey, @swfz.

No, unfortunately, you can't. 

Static resources like images and iframes are not loaded in a way to trigger the worker's "fetch" event. Because of this, MSW doesn't know that any requests are happening and cannot influence them in any way.
